contracts/ERC777/ERC777BaseToken.sol

```diff
@@ -137,16 +150,6 @@ contract ERC777BaseToken is ERC777Token, ERC820Client {
     require(_amount.div(mGranularity).mul(mGranularity) == _amount);
   }
 
-  /// @notice Check whether an address is a regular address or not.
-  /// @param _addr Address of the contract that has to be checked
-  /// @return `true` if `_addr` is a regular address (not a contract)
-  function isRegularAddress(address _addr) internal constant returns(bool) {
```

Comment:
I dont think code removal/addition is proper when the branch only aims for Solium & linting stuff. But it's okay for now.

Reply:
I removed the code because It triggers a security warning called "security/no-inline-assembly".
I agree that it's not necessary to remove the code only to avoid a warning, but since there's `AddressUtils` contract in openzeppelin-solidity doing the same thing, I think it's better we import `AddressUtils` and make our code cleaner.